developer/decision-records/2023-09-06-identity-trust/README.md

```diff
@@ -0,0 +1,105 @@
+maintained in the [Eclipse Tractus-X SSI repository](https://github.com/eclipse-tractusx/ssi-docu). These specifications
+will be transferred to the Eclipse Dataspace Working Group as
+described [here](#protocol-specifications-and-the-eclipse-dataspace-working-group).
+
+### Self-Issued Identity Tokens
+
+The shape of self-issued identity tokens and protocol for obtaining them will be based on
+the [Self-Issued OpenID Provider v2 specification](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html#section-1.1) (
+SIOPv2).
+
+### Verifiable Credential Presentation
+
+Data providers will often require a set of Verifiable Presentations (i.e. the presentation of a Verifiable Credential)
+to access their assets. In addition, trust anchors such as Credential Issuers need a way to deliver Verifiable
+Credentials to a holder.
+
+The Verifiable Presentation protocol will define a set of operations for reading and writing Verifiable Credentials
+using the W3C [Verifiable Credentials Data Model v1.1](https://www.w3.org/TR/vc-data-model/). The protocol will also
+make use of
+the [DIF Presentation Exchange specification](https://identity.foundation/presentation-exchange/spec/v2.0.0/).
+
+Note that
+the [OpenID for Verifiable Presentations specification](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-6.1)
+was considered, but determined not to be sufficient, due to its focus on end-user as opposed to service-to-service
+interactions. [DidComm Messaging](https://identity.foundation/didcomm-messaging/spec/) was also investigated as a
+potential starting point but not selected due to unneeded complexity and lack of widespread adoption.
+
+### Verifiable Credential Issuance
+
+Dataspaces will require an interoperable way for participant agent systems to request credential issuance, renew
+credentials, and check for credential revocation. The Verifiable Credential Issuance will define a RESTful API over
+HTTPS that uses the [W3C Decentralized Identifiers specification](https://www.w3.org/TR/did-core/) and
+the [W3C Status List specification](https://www.w3.org/TR/vc-status-list/).
+
+Note that
+the [OpenID for Verifiable Credential Issuance specification](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-introduction)
+was considered, but determined not to be sufficient, due to its focus on end-user interactions and lack of support for
+long-running issuance processes that may involve human interaction.
+
+### Protocol Specifications and The Eclipse Dataspace Working Group
+
+The protocol specifications (i.e. documents that define the message exchanges) for the Identity and Trust System will be
+transferred to the [Eclipse Dataspace Working Group](https://www.eclipse.org/org/workinggroups/dataspace-charter.php)
+for standardization. The EDC will track changes to the protocol specifications as they undergo standardization.
+
+## Impact
```

Comment:
```suggestion
### Adoption in EDC
```

Reply:
This section should be impact as that is what it is describing